src/os/bluestore/bluestore_tool.cc

```diff
@@ -219,6 +227,48 @@ void inferring_bluefs_devices(vector<string>& devs, std::string& path)
   }
 }
 
+void bluefs_import(
+  string input_file,
```

Comment:
For the sake of convenience please provide both input_file and destination_file.